regcore_write/views/preamble.py

```diff
@@ -0,0 +1,16 @@
+import json
+
+from regcore.db import storage
+from regcore.responses import success, user_error
+from regcore_write.views.security import secure_write
+
+
+@secure_write
+def add(request, docnum):
+    """Add the preamble to the db"""
+    try:
+        preamble = json.loads(request.body.decode('utf-8'))
+    except (ValueError, UnicodeError):
```

Comment:
Another drive-by question. I don't know how Django handles this kind of thing, but it would be neat if we could move this repeated logic into a helper function that throws an exception if we can't parse, and that exception is caught by Django and turned into a 400 or 422 or whatever. It looks like we could raise `SuspiciousOperation`, since that gets translated into a 400, but that might not be semantically correct.

Basically, I'm looking for the Django equivalent of http://flask.pocoo.org/docs/0.10/patterns/apierrors/#registering-an-error-handler.